tests/testthat/test_post_processing.R

```diff
@@ -0,0 +1,75 @@
+context("post processing")
+
+test_that("Post processing yields to integer sample sizes", {
+    # define an initial design
+    order <- 5L # must be integer!
+
+    design <- gq_design(
+        n1 = 100,
+        c1f = 0,
+        c1e = 2,
+        n2 = 10 * log(seq(600.0, 100.0, length.out = order)), #use something not too smooth
+        c2 = rep( 1.96, order),
+        order = order)
+
+    # Try a continuous prior
+    null        <- PointMassPrior(.0, 1)
+    dichte_alt  <- function(x) dnorm(x, mean = .3, sd = .1)
+    alternative <- ContinuousPrior(dichte_alt, c(-5, 5))
```

Comment:
why not use ContinuousPrior(dichte_alt, c(-1, 2)) ?

Reply:
This is a solution for this specific problem
But I think with another prior there could be again problems and it should not be the goal that the user has to look for the fitting support for every new prior, see my comment above